opt/cs50/bin/rstudio

```diff
@@ -1,12 +1,25 @@
 #!/bin/bash
 
-IMAGE="cs50/rstudio"
+IMAGE="rocker/tidyverse"
```

Comment:
We are now building our own [cs50/rstudio image](https://github.com/cs50/rstudio) just so we can have more control over RStudio's experience (e.g., submit50/check50)

Reply:
oops, overzealous copy/paste, fixed